airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -127,6 +141,30 @@ public JsonNode toJdbcConfig(JsonNode config) {
                                                                              JdbcStateManager stateManager,
                                                                              Instant emittedAt) {
     if (isCdc(config)) {
+      final String targetLsn = getLsn(database);
+      LOGGER.info("identified target lsn: " + targetLsn);
+      final Predicate<JsonNode> hasReachedLsnPredicate = (record) -> Optional.ofNullable(record.get("value"))
```

Comment:
what is the schema of the object being deserialized? is that mentioned anywhere?